program/convert-ssd-to-i8/README.md

```diff
@@ -0,0 +1,52 @@
+# convert-ssd-to-i8
+
+Program for conversion of SSD models into Intel Caffe quantized format.
+
+## Requirements
+
+Intel Caffe library:
+```
+ck install package --tags=lib,caffe,intel
+```
+
+Caffe model to be converted into quantized format:
+```
+ck install package --tags=caffemodel,ssd
+```
+
+One of datasets COCO-2014 or VOC-2207 depending on wich model you want convert:
+```
+ck install package:dataset-coco-2014
+ck install package:dataset-voc-2007
+```
+
+## Run
+
+```
+ck run program:convert-ssd-to-i8 --cmd_key=convert-coco
+```
+or
+```
+ck run program:convert-ssd-to-i8 --cmd_key=convert-voc
+```
+
+Program takes dataset and prepares LMDB database. Number of images to be included into database is governed by variable `CK_IMAGES_PERCENT`.
+
+Resulting models are stored into `tmp` directory as `test_quantized.prototxt` and `deploy_quantized.prototxt`.
+
+## Test
+
+Program also provide two testing commands:
+```
+ck run program:convert-ssd-to-i8 --cmd_key=test-f32
+ck run program:convert-ssd-to-i8 --cmd_key=test-i8
+```
+Commands take LMDB database that was prepared at previous `convert` stage and run `caffe test` operaion using original or converted models.
```

Comment:
`caffe test` -> `ck run program:caffe --test_cpu` ?

Reply:
No, it does not run `ck run program:caffe`, it runs
```
${CK-TOOLS}/lib-caffe-*/install/bin/caffe test --model=... --weights=... --iterations=... --detect
```